docs/en/01_basic-configuration.md

```diff
@@ -1,7 +1,13 @@
-# Basic Configuration
+---
+title: Basic Configuration
```

Comment:
```suggestion
title: Basic Static Publishing Configuration
```

Reply:
I don't agree with this - see https://github.com/silverstripe/silverstripe-realme/pull/192#discussion_r2218139490